<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QWidget" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageUserInfo">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,1">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="labelTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Registration</string>
           </property>
           <property name="margin">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>40</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelName">
             <property name="text">
              <string>Full Name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelTaxNumber">
             <property name="text">
              <string>Tax Number</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelIncome">
             <property name="text">
              <string>Income</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelCardType">
             <property name="text">
              <string>Card Type</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="editName"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="editPhone"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="editTaxNumber"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="editIncome"/>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="comboCardType"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelPhone">
             <property name="text">
              <string>Phone</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>|Enter| Submit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>|Cancel| Back</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSuccessfulRegistration">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="labelPin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Your pin: 1234</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnTakeCard">
           <property name="text">
            <string>|Card| Take Your New Card</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
